tests/integration/tier0/custom-kernel/test_cutsom_kernel.py

```diff
@@ -0,0 +1,18 @@
+import platform
+
+from envparse import env
+
+
+def test_custom_kernel(convert2rhel):
+    # Run c2r with --variant option
```

Comment:
```suggestion
    # Run c2r with --variant option
```
Isn't this a copy paste error?